nmrfx-analyst-gui/src/main/java/org/nmrfx/processor/gui/GUI/ModifiableAccordionScrollPane.java

```diff
@@ -0,0 +1,118 @@
+            // Create spacer to separate label and buttons
+            Region spacer = new Region();
+            HBox.setHgrow(spacer, Priority.ALWAYS);
+            titleBox.getChildren().add(spacer);
+            // Create Right Aligned Buttons
+            Button upButton = GlyphsDude.createIconButton(FontAwesomeIcon.ARROW_UP, "", AnalystApp.ICON_SIZE_STR, AnalystApp.REG_FONT_SIZE_STR, ContentDisplay.GRAPHIC_ONLY);
+            upButton.setOnAction(this::moveUp);
+            Button downButton = GlyphsDude.createIconButton(FontAwesomeIcon.ARROW_DOWN, "", AnalystApp.ICON_SIZE_STR, AnalystApp.REG_FONT_SIZE_STR, ContentDisplay.GRAPHIC_ONLY);
+            downButton.setOnAction(this::moveDown);
+            Button closeButton = GlyphsDude.createIconButton(FontAwesomeIcon.CLOSE, "", AnalystApp.ICON_SIZE_STR, AnalystApp.REG_FONT_SIZE_STR, ContentDisplay.GRAPHIC_ONLY);
+            closeButton.setOnAction(this::removeTitledPane);
+            titleBox.getChildren().addAll(downButton, upButton, closeButton);
+            return titleBox;
+        }
+
+        private void moveUp(ActionEvent actionEvent) {
+            int currentIndex = getIndexOfTitledPane((Button) actionEvent.getSource());
+            int newIndex = Math.max(currentIndex - 1, 0);
+            TitledPane tp = getPanes().remove(currentIndex);
+            getPanes().add(newIndex, tp);
+        }
+
+        private void moveDown(ActionEvent actionEvent) {
+            int currentIndex = getIndexOfTitledPane((Button) actionEvent.getSource());
+            int newIndex = Math.min(currentIndex + 1, getPanes().size() - 1);
+            TitledPane tp = getPanes().remove(currentIndex);
+            getPanes().add(newIndex, tp);
+        }
+
+        private void removeTitledPane(ActionEvent actionEvent) {
+            getPanes().remove(getIndexOfTitledPane((Button) actionEvent.getSource()));
+        }
+
+        private int getIndexOfTitledPane(Button button) {
+            Optional<TitledPane> paneToRemove = getPanes().stream().filter(titledPane -> ((HBox) titledPane.getGraphic()).getChildren().contains(button)).findFirst();
+            return paneToRemove.map(titledPane -> getPanes().indexOf(titledPane)).orElse(-1);
+        }
+    }
+
+    private static class ButtonTitlePaneSkin extends TitledPaneSkin {
+        final Region arrow;
+
+        ButtonTitlePaneSkin(final TitledPane titledPane) {
+            super(titledPane);
+            arrow = (Region) getSkinnable().lookup(".arrow-button");
+
+        }
+
+        @Override
+        protected void layoutChildren(final double x, final double y, final double w, final double h) {
+            super.layoutChildren(x, y, w, h);
+            double arrowWidth = arrow.getLayoutBounds().getWidth();
+            double arrowPadding = arrow.getPadding().getLeft() + arrow.getPadding().getRight();
+
+            ((Region) getSkinnable().getGraphic()).setMinWidth(w - (arrowWidth + arrowPadding));
```

Comment:
moved what was in the listener/property binding here instead. This is much neater and seems to work well. It was still not my idea solution since it only works with ContentDisplay.GRAPHIC_ONLY, meaning I need to add a label for the title instead of using the built in text. I did attempt to calculate the length of the gap between the text and the graphic to be able to call setGraphicTextGap but I could not get the calculations to work at all, so decided to just use the GRAPHIC_ONLY display